localstack_snapshot/snapshots/report.py

```diff
@@ -29,6 +30,8 @@
     "underlined": 4,
 }
 
+_special_json_path_chars_regex = re.compile("[a-zA-Z0-9_-]+")
```

Comment:
suggestion: rename variable to match (pun intended 😉) the regex purpose.
```suggestion
_regular_json_path_chars_regex = re.compile("[a-zA-Z0-9_-]+")
```

Reply:
Oops, good point 😄 thanks!

Reply:
This will allow the condition below to be more understandable: now it says "if doesn't match special characters then add quotes" while in reality it should be "if doesn't match regular characters then add quotes"